src/binarySearch.js

```
@@ -0,0 +1,28 @@
+export default function binarySearch( arr, query){
+  if( arr == null ){
+    return 'Cannot search null'
```

I think you should raise errors here instead of return an error message as a string